src/AwesomeAlert.js

```diff
@@ -103,9 +107,9 @@ export default class AwesomeAlert extends Component {
     } = data;
 
     return (
-      <TouchableOpacity  style={[styles.button, { backgroundColor }, buttonStyle]} testID={testID} onPress={onPress}>
+        <TouchableOpacity  style={[styles.button, { backgroundColor }, buttonStyle]} testID={testID} onPress={onPress}>
```

Comment:
there's still an extra space here that makes this change show up in the diff of the PR. If you remove that extra spacing, this would not show up here. Thats the state we'd want to be in.
```suggestion
      <TouchableOpacity  style={[styles.button, { backgroundColor }, buttonStyle]} testID={testID} onPress={onPress}>
```